docker-compose.yml

```diff
@@ -795,7 +795,7 @@ services:
     hostname: appwrite-executor
     <<: *x-logging
     stop_signal: SIGINT
-    image: openruntimes/executor:0.5.1
+    image: openruntimes/executor:0.5.2
```

Comment:
@vermakhushboo the executor version needs to be updated in the compose.phtml file as well as in the cloud repo